README.md

```diff
@@ -8,13 +8,50 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](#Getting-Started)  
+    - [Eclipse Plugin Installation Instuctions](#Eclipse-Plugin-Installation-Instuctions)
+- [Building](#Building)
+    - [Prerequisites](#Prerequisites)
+    - [Setup Instructions](#Setup-Instructions)
+    - [Common Errors](#Common-Errors)
+- [Contributing](#Contributing)
+- [Feedback](#Feedback)
+---
 ## Getting Started
 
+### Eclipse Plugin Installation Instuctions 
```

Comment:
We can remove this header and instead add a sentence indicating that the following instructions are for installing the plugin  to an Eclipse instance